pgjdbc/build.gradle.kts

```diff
@@ -188,10 +188,10 @@ tasks.compileJava {
 val getShadedDependencyLicenses by tasks.registering(GatherLicenseTask::class) {
     configuration(shaded)
     extraLicenseDir.set(file("$rootDir/licenses"))
-    overrideLicense("com.ongres.scram:common") {
+    overrideLicense("com.ongres.scram:scram-common") {
         licenseFiles = "scram"
     }
-    overrideLicense("com.ongres.scram:client") {
+    overrideLicense("com.ongres.scram:scram-client") {
         licenseFiles = "scram"
     }
```

Comment:
If `scram-*` bundles the license file, we could probably remove customizations here, and remove `/licenses/*` files.
WDYT?

Reply:
Yes, it bundle the license now, not sure what exact does Gradle here, so if it's not needed to do anything else here, it could be probably removed.

Reply:
The configuration here sets to use license files from https://github.com/pgjdbc/pgjdbc/tree/master/licenses/scram for the scram library. Ideally, libraries should bundle their licenses (and the licenses of the shaded libraries as well), so in the ideal world, overrides should not be needed. However, previous scram versions missed the license files so we had to keep them in pgjdbc source tree

Reply:
Ouch!, I just checked that the stringprep dependency does not have bundled the LICENSE file, was completely an oversight, so we might need to keep the overrides a bit longer.

Reply:
Will need some help here, should I comment on all the `overrideLicense` blocks?
Should I add something more to make it work?
Right now if I comment all the `overrideLicense` blocks, the output is just a single dependency, what else needs to be done?